lib/jekyll-paginate-v2/generator/paginationModel.rb

```diff
@@ -137,7 +137,7 @@ def _fix_deprecated_config_features(config)
         end
 
         # Delete the depricated keys
-        config.delete_if{ |k,| keys_to_delete.include? k }
+        config.delete_if { |k, _v| keys_to_delete.include? k }
```

Comment:
I left the second variable out because it is clearer that it is not being used that way

Reply:
Did you know that you could use the "suggestions" feature here to make the changes for me..?
Nevertheless, I'll make the change locally and push a single commit to avoid "churn" in the git history..